taskt/Core/Project.cs

```diff
@@ -88,7 +88,7 @@ public static Project OpenProject(string mainFilePath)
                 foreach (string scriptFullPath in scriptFullPaths)
                 {
                     string relativeScriptPath = scriptFullPath.Replace(projectPath, "");
-                    if (relativeScriptPath.Contains(".xml"))
+                    if (relativeScriptPath.Contains(".json"))
```

Comment:
So what this does is looks for all scripts in the project directory and adds them to the project.json if they have a file extension of .xml. By changing this to .json, the project.json will include itself as a script in its list of scripts. Therefore we're definitely going to need a new extension for the project.json.

Reply:
I'll add a check that the path does not contain project.json.